scripts/prepare_for_nuget_pack.sh

```diff
@@ -65,29 +39,29 @@ cd $PRODUCTS_DIR
 
 # Copy each platform
 make_dir_if_necessary "nuget/Debug-macosx"
-rsync_excluding_binary_swift_module_files_framework Debug/FluentUI.framework/ nuget/Debug-macosx/FluentUI.framework/
+rsync -a Debug/FluentUI.framework/ nuget/Debug-macosx/FluentUI.framework/
```

Comment:
is the space before NuGet intentional?

Reply:
Yeah `Debug/FluentUI.framework/` is the source of the copy, `nuget/Debug-macosx/FluentUI.framework/` is the destination